### ISSUE TITLE
stratum x13

### ISSUE THREAD

**Issue by goodthebest** [eveybcd/x13bcd_hash#3]
@heyconnor @Starry9 I'm trying to integrate bcd enhanced x13 into yiimp, in stratum.h there is existing value for x13. Just wondering if values are okay or need to change?

`{"x13", x13_hash, 1, 0, 0},`

**Comment by Starry9** [comment]
The x13bcd_hash is different from x13_hash.
I suggest you can edit x13_hash in yiimp (https://github.com/tpruvot/yiimp/blob/next/stratum/algos/x13.c) by example with x13bcd_hash (https://github.com/eveybcd/x13bcd_hash/blob/master/x13.c) if you know a little about C language

**Comment by goodthebest** [comment]
I've done that already but **x13.c** isn't the only file needs to be done, there are **x13.h** as well as all files **sha3** needs to be worked on, right?

**Comment by Starry9** [comment]
Yes, edit x13.h if you need, and copy sha3/sph_sm3.h to sha3/ in yiimp.
